Laba2SCCS/src/BusinessLayer/Interfaces/IService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Interfaces
+{
+    public interface IService<TModel>
+        where TModel : class
+    {
+        Task<int> Create(TModel item);
```

Comment:
I guess instead of int would be better to have generic here like:
```csharp
public interface IService<TModel> where TModel : IIdentity<T> where T: int
{
    Task<T> Create(TModel item);
    Task<TModel> GetById(T id);
    ....
}
```

It means both models categories (DAL and BLL) should implement an interface
```csharp
public interface IIdentity<in T>
{
    T Id { get; }
}
```

It allows us to avoid errors with different types of keys and have consistency.